process/process.lua

```diff
@@ -367,11 +370,20 @@ function process.handle(msg, _)
       function (msg)
         return msg.Tags.Type == "Process" and Owner == msg.From 
       end,
-      require('.boot')(ao)
+      function ()
+        require('.boot')(ao)    
+        -- AR.IO PROCESS CODE
+        require(".src.init").init()
+        -- load all the initial state once
+        require(".state.init").init()
+      end
     )
   end
 
-  Handlers.append("_default", function () return true end, require('.default')(insertInbox))
+  Handlers.append("_default", function () return true end, function (msg)
+    print('{ "_e": 1, "Message-Id": "' .. msg.Id .. '", "From": "' .. msg.From .. '", "Action": "' .. msg.Action .. '", "Timestamp": ' .. tonumber(msg.Timestamp) .. ', "Default-Handler": true}')
+    return require('.default')(insertInbox)(msg)
```

Comment:
Same question here about the need for return.

Reply:
not sure if it's necesasry - feel free to test by removing

Reply:
a3f3523